DBZ-7917: Document PostgreSQL read-only incremental snapshot. (#5624)
debezium/debezium: documentation/modules/ROOT/pages/connectors/postgresql.adoc

include::{partialsdir}/modules/all-connectors/proc-stopping-an-incremental-snapshot-kafka.adoc[leveloffset=+1]

ifdef::community[]
[id="postgresql-read-only-incremental-snapshots"]
==== Read-only incremental snapshots

> The PostgreSQL connector allows for running incremental snapshots with a read-only connection to the database.

Reviewer: ```suggestion
You can configure a PostgreSQL connector that has a read-only connection to the database to run incremental snapshots.
```
I'm in the process of updating the MySQL/MariaDB docs, and will apply similar wording.

Author: I just update it adding 'without requiring the signal data collection table.' WDYT?

Reviewer: Changed it to "a signal data collection table". I think that works, si?

To run an incremental snapshot with read-only access, the connector uses the current in-progress transaction as high and low watermarks.

> The state of a chunk's window is updated by comparing the txId of write-ahead-log events or the heartbeats against low and high watermarks.

Reviewer: ```suggestion
The state of a chunk's window is updated by comparing the transaction ID of write-ahead-log events or heartbeat events against low and high watermarks.
```

No entry is created for the signal to close the snapshot window.
Set this option to prevent rapid growth of the signaling data collection.

ifdef::community[]
|[[postgres-property-read-only]]<<postgres-property-read-only, `+read.only+`>>
|`false`

> |Switch to alternative incremental snapshot watermarks implementation to avoid writes to signal data collection

Reviewer: ```suggestion
|Specifies whether a connector writes watermarks to the signal data collection to track the progress of an incremental snapshot.
Set the value to `true` to enable a connector that has a read-only connection to the database to use an incremental snapshot watermarking strategy that does not require writing to the signal data collection.
```
I've applied the same wording to the MySQL/MariaDB docs.
